excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/RoleToAccessProfilesParser.java

```diff
@@ -52,16 +53,17 @@ public RoleToAccessProfilesEntity createRoleToAccessProfileEntity(ParseContext p
                 throw new InvalidImportException(message);
 
             });
-        roleToAccessProfilesEntity.setCaseType(caseTypeEntity);
 
+        roleToAccessProfilesEntity.setCaseType(caseTypeEntity);
         roleToAccessProfilesEntity.setRoleName(definitionDataItem.getString(ColumnName.ROLE_NAME));
         roleToAccessProfilesEntity.setLiveFrom(definitionDataItem.getDate(ColumnName.LIVE_FROM));
         roleToAccessProfilesEntity.setLiveTo(definitionDataItem.getDate(ColumnName.LIVE_TO));
         roleToAccessProfilesEntity.setReadOnly(definitionDataItem.getBooleanOrDefault(ColumnName.READ_ONLY, false));
         roleToAccessProfilesEntity.setDisabled(definitionDataItem.getBooleanOrDefault(ColumnName.DISABLED, false));
-        String accessProfiles = definitionDataItem.getString(ColumnName.ACCESS_PROFILES);
-        roleToAccessProfilesEntity.setAccessProfiles(accessProfiles);
+        roleToAccessProfilesEntity.setAccessProfiles(definitionDataItem.getString(ColumnName.ACCESS_PROFILES));
         roleToAccessProfilesEntity.setAuthorisation(definitionDataItem.getString(ColumnName.AUTHORISATION));
+        final String categories = definitionDataItem.getString(ColumnName.CASE_ACCESS_CATEGORIES);
+        roleToAccessProfilesEntity.setCaseAccessCategories(categories);
```

Comment:
Can just set it straight away like the `accessProfiles` was changed to a few lines up?

Reply:
Actioned in e2aa9add04f3f367beedea20683c03decc7a5716